### PR TITLE
11102022 as import ad group users to ms watchlist

### PR DESCRIPTION
**Pull request by AcceleryntSecurityDev**
Required items, please complete
   
   Change(s):
   - New Playbook

   Reason for Change(s):
   - New Playbook

   Testing Completed:
  - Yes

Issues:
  - I have been implementing all the recommended contribution guidelines on previous submissions without issue, however, the recommend sentinel "ManagedSecurityIdentity" connection seems incompatible with the sentinel watchlist connector. I have had no issue using managed identity connections with other sentinel triggers and connectors in previous deployment templates, so I suspect the issue is with the connector. Because of this, I have intentionally reverted the sentinel connector back to the previous version (not managed identity), since I did not want to publish something with a broken feature. I'm hoping this issue can be escalated and fixed on the MS side.

Here's a description of what happened when I deployed the template with the managed service identity:
After validating the azure ad connection and running the playbook, the run failed on the sentinel watchlist connector with the managed identity connection created at deployment. The error message suggested the path to the watchlist resource was incorrect. However, when I changed nothing but the connection from the managed identity created at deployment to an existing sentinel connection, the playbook ran successfully. When I tried to switch back to the managed identity connector to troubleshoot further, it no longer showed up in the list of available connections for the sentinel watchlist connector, though I could see that it still exited in the resource group.

### CODE REVIEW

**Comment by v-spadarthi** [comment]
@rahul0216 / @manishkumar1991 : Please review the playbooks and provide your comments. Thanks!

**Comment by v-spadarthi** [comment]
@rahul0216 / @manishkumar1991 : Please review the playbooks and provide your comments. Thanks!

**Review by manishkumar1991 — COMMENTED** [review]
@AcceleryntSecurityDev ,

Provided sentinel connection should be "ManagedServiceIdentity "

"connectionProperties": {
"authentication": {
"type": "ManagedServiceIdentity"
}
}

Please check below playbook for reference:
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/OpenCTI/Playbooks/OpenCTIPlaybooks/OpenCTI-EnrichIncident/azuredeploy.json

**Comment by v-spadarthi** [comment]
@AcceleryntSecurityDev : Please address the @manishkumar1991 comments. Thanks!

**Comment by v-spadarthi** [comment]
@AcceleryntSecurityDev : Please address the @manishkumar1991 comments. Thanks!

**Comment by v-spadarthi** [comment]
@AcceleryntSecurityDev : Please address the @manishkumar1991 comments. Thanks!

**Comment by AcceleryntSecurityDev** [comment]
@manishkumar1991 I have updated the sentinel connection to use managed identity, though I intentionally neglected to do this because of the issue listed above. Please let me know if you run into the same trouble.

**Review by rahul0216 — COMMENTED** [review]
1. Please address the inline comments.
2. Playbook requires at least Microsoft Sentinel Contributor role on Log Analytics workspace to perform the actions. Please add steps to add this role to LA workspace.

File: Playbooks/AS-Import-AD-Group-Users-to-MS-Watchlist/README.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+Click on the “**Edit**” button. This will bring us into the Logic Apps Designer.
+
+![AS_Group_Watchlist_Deploy_4](Images/AS_Group_Watchlist_Deploy_4.png)
+
+Before the playbook can be run successfully, the Azure AD connection used in the second step and the Microsoft Sentinel connection used in the fourth and ninth steps will either need to be authorized, or existing authorized connections may be alternatively selected.  
+
+![AS_Group_Watchlist_Deploy_5](Images/AS_Group_Watchlist_Deploy_5.png)
+
+To validate the Azure AD connection, expand the second step labeled "**Connections**" and click the exclamation point icon next to the name matching the playbook.
```
[the remark this answers is not in the document]
Comment (AcceleryntSecurityDev) [reply Playbooks/AS-Import-AD-Group-Users-to-MS-Watchlist/README.md:137]: image
I think the spelling does not need to be changed.

[the remark this answers is not in the document]
Comment (rahul0216) [reply Playbooks/AS-Import-AD-Group-Users-to-MS-Watchlist/README.md:137]: image
Interesting, American vs British English. Looks like both are correct, no need to update.

**Comment by v-spadarthi** [comment]
@AcceleryntSecurityDev: Please address the @rahul0216 comments. Thanks!

**Review by rahul0216 — COMMENTED** [review]
Playbook requires at least Microsoft Sentinel Contributor role on Log Analytics workspace to perform the actions. Please add steps to add this role to LA workspace. You can check this readme for help, https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Australian%20Cyber%20Security%20Centre/Playbooks/AusCtisExportTaggedIndicators/readme.md
Check the Post Deployment instruction "a". These steps are to provide Contributor Role, but replace it with Microsoft Sentinel Contributor, for least privileged.

**Comment by v-spadarthi** [comment]
@AcceleryntSecurityDev : Please address the @rahul0216 comments.

**Comment by AcceleryntSecurityDev** [comment]
@rahul0216 https://github.com/Accelerynt-Security/Azure-Sentinel/tree/11102022-AS-Import-AD-Group-Users-to-MS-Watchlist/Playbooks/AS-Import-Azure-AD-Group-Users-to-MS-Watchlist#add-microsoft-sentinel-contributor-role

**Review by rahul0216 — APPROVED** [review]
Thank you. Changes look good.

**Review by v-mchatla — APPROVED** [review]

**Review by v-rucdu — APPROVED** [review]

**Pull request merged**